packages/react-migration-v8-v9/src/components/Theme/v8ThemeShim.ts

```diff
@@ -111,113 +135,46 @@ const mapPalette = (brandColors: BrandVariants, inverted: boolean): IPalette =>
-    inputBorder: theme.colorNeutralStrokeAccessible,
-    inputBorderHovered: theme.colorNeutralStrokeAccessibleHover,
-    inputFocusBorderAlt: theme.colorBrandStroke1,
-    inputForegroundChecked: theme.colorNeutralForegroundOnBrand,
-    inputIcon: theme.colorNeutralForeground3,
-    inputIconDisabled: theme.colorNeutralForegroundDisabled,
-    inputIconHovered: theme.colorNeutralForeground3,
-    inputPlaceholderBackgroundChecked: theme.colorBrandBackgroundInvertedSelected,
-    inputPlaceholderText: theme.colorNeutralForeground4,
-    inputText: theme.colorNeutralForeground1,
-    inputTextHovered: theme.colorNeutralForeground1Hover,
-    link: theme.colorBrandForegroundLink,
-    linkHovered: theme.colorBrandForegroundLinkHover,
-    listBackground: theme.colorNeutralBackground1,
-    listHeaderBackgroundHovered: theme.colorNeutralBackground1Hover,
-    listHeaderBackgroundPressed: theme.colorNeutralBackground1Pressed,
-    listItemBackgroundChecked: theme.colorNeutralBackground1Selected,
-    listItemBackgroundCheckedHovered: theme.colorNeutralBackground1Selected,
-    listItemBackgroundHovered: theme.colorNeutralBackground1Hover,
-    listText: theme.colorNeutralForeground1,
-    listTextColor: theme.colorNeutralForeground1,
-    menuBackground: theme.colorNeutralBackground1,
-    menuDivider: theme.colorNeutralStroke2,
-    menuHeader: theme.colorNeutralForeground3,
-    menuIcon: theme.colorNeutralForeground1,
-    menuItemBackgroundChecked: theme.colorNeutralBackground1,
-    menuItemBackgroundHovered: theme.colorNeutralBackground1Hover,
-    menuItemBackgroundPressed: theme.colorNeutralBackground1Hover,
-    menuItemText: theme.colorNeutralForeground1,
-    menuItemTextHovered: theme.colorNeutralForeground1Hover,
-    messageLink: theme.colorBrandForegroundLink,
-    messageLinkHovered: theme.colorBrandForegroundLinkHover,
-    messageText: theme.colorNeutralForeground1,
-    primaryButtonBackground: theme.colorBrandBackground,
-    primaryButtonBackgroundDisabled: theme.colorNeutralBackgroundDisabled,
-    primaryButtonBackgroundHovered: theme.colorBrandBackgroundHover,
-    primaryButtonBackgroundPressed: theme.colorBrandBackgroundPressed,
-    primaryButtonBorder: theme.colorTransparentStroke,
-    primaryButtonText: theme.colorNeutralForegroundOnBrand,
-    primaryButtonTextDisabled: theme.colorNeutralForegroundDisabled,
-    primaryButtonTextHovered: theme.colorNeutralForegroundOnBrand,
-    primaryButtonTextPressed: theme.colorNeutralForegroundOnBrand,
-    // severeWarningBackground,
-    // severeWarningIcon,
-    // smallInputBorder,
-    // successBackground,
-    // successIcon,
-    // successText: ,
-    // variantBorder,
-    // variantBorderHovered,
-    // warningBackground,
-    // warningHighlight,
-    // warningIcon,
-    // warningText: ,
-  };
+const mapSemanticColors = (baseColors: ISemanticColors, v8Palette: IPalette, isDarkTheme: boolean): ISemanticColors => {
+  const p = v8Palette;
```

Comment:
This file probably needs special consideration. The user could update the brand ramp, but it wouldn't trickle to the rest of the components via the palette->semanticSlot mappings in v8.

The big con here is that we're effectively no-opping the v9 theme other than the brand ramp.

Pro is that we'll likely see more alignment on the neutrals because teams will have to manually modify anything beyond the brand.